libqtile/backend/wayland/core.py

```diff
@@ -56,6 +57,10 @@
     OutputConfigurationV1,
     OutputManagerV1,
 )
+from wlroots.wlr_types.pointer_constraints_v1 import (
+    PointerContraintsV1,
```

Comment:
Is this a typo in the `wlroots` module? I would have thought this should be `PointerConstraintsV1`...

Reply:
Brilliant, the typo is also in pywlroots

Reply:
https://github.com/flacjacket/pywlroots/pull/59